src/autopas/utils/SimilarityFunctions.h

```diff
@@ -0,0 +1,106 @@
+/**
+ * @file SimilarityFunctions.h
+ * @author J. Kroll
+ * @date 01.06.2020
+ */
+
+#pragma once
+
+#include "ThreeDimensionalMapping.h"
+#include "autopas/containers/ParticleContainerInterface.h"
+#include "autopas/molecularDynamics/MoleculeLJ.h"
+
+namespace autopas::utils {
+
+/**
+ * calculates homogeneity and max density of given AutoPas simulation
+ * homogeneity > 0.0, normally < 1.0, but for extreme scenarios > 1.0
+ * maxDensity > 0.0, normally < 3.0, but for extreme scenarios >> 3.0
+ *
+ * @tparam Particle
+ * @param container container of current simulation
+ * @return {homogeneity, maxDensity}
+ */
+template <class Particle>
+std::pair<double, double> calculateHomogeneityAndMaxDensity(
```

Comment:
this should probably replace the homogeneity calculation in md-flex to avoid code duplication.

Reply:
resolved by commit 4644b904f47f6974575b899758eae53e8d4318d2
needed to rename a member function of AutoPas

Reply:
renamed all instances of getNumParticles to getNumberOfParticles in commit a7e972230da0c13e1ff5f11f053d6980167bf28b
had to redo some after merging from master in commit 8566c1a89ed584b83843b19ad89a9d74992e763e